client/src/elm/Measurement/View.elm

```diff
@@ -3158,6 +3104,178 @@ ncdaFormInputsAndTasks language currentDate personId person config form currentS
             )
 
 
+ancVisitsInpustAndTasks :
+    Language
+    -> NominalDate
+    -> PersonId
+    -> Person
+    -> NCDAContentConfig msg
+    -> NCDAForm msg
+    -> ModelIndexedDb
+    -> ( List (Html msg), List (Maybe Bool) )
+ancVisitsInpustAndTasks language currentDate personId person config form db =
+    let
+        encountersDatesFromANCData =
+            resolveChildANCEncountersDates personId db
+    in
+    if EverySet.size encountersDatesFromANCData >= 4 then
+        ( [], [] )
+
+    else
+        let
+            encountersDatesFromForm =
+                -- Since ANC step of NCDA form is filled only onces, we know
+                -- that current activity is the first one filled, and there's
+                -- no need to examine existing NCDA activities.
```

Comment:
```suggestion
                -- Since ANC step of NCDA form is filled only once, we know
                -- that current activity is the first one filled, and there's
                -- no need to examine existing NCDA activities.
```